### DISCUSSION TITLE
Approved as faculty instead of as a student.

### DISCUSSION THREAD

**Discussion by LjeNuiN** [community/community#19314] [GitHub Education] [answered]
I am approved as a faculty and unable to access any of the benefits in the student developer pack

**Comment by melvincwng** [comment, marked as answer]
@LjeNuiN, reapply again for the Student Developer Pack at Github Education.

Hope this clarifies.

**Reply by melvincwng** [reply]
@LjeNuiN, if this reply is helpful, please 'Mark as answer'. Thank you.

**Reply by LjeNuiN** [reply]
@melvincwng  I am unable to apply again because I already applied when I tried to apply the submit button is grayed out and it also tells me there's no need to reapply cuz I'm already verified and when I try to apply to the student developer pack it just redirects me to my application that was approved for the faculty

**Reply by melvincwng** [reply]
@LjeNuiN, in that case, please kindly raise a ticket and contact [Github Support](https://support.github.com/contact/account).

They will investigate and get back to you within a couple of days.

Hope this clarifies.

**Reply by Meeta-motherfds** [reply]
Same happened with me also. What I do now?

**Reply by melvincwng** [reply]
@Meeta-motherfds, try contacting github support and let them investigate.

**Reply by LjeNuiN** [reply]
I contacted support but my ticket was just closed with no feedback. I tried reopening it. I even tried downgrading to free from faculty so now i dont even have pro at least. Hopefully i get some resolution. How are we supposed to become used to the tools to the point we wanna pay for them if we cant even try them when we are in school lol. Fingers crossed i dont get auto closed again.

**Reply by LjeNuiN** [reply]
@melvincwng @Meeta-motherfds  My issue has finally been resolved. They did something on their end and I was able to apply again even though I had an approved status. But then I was presented with more issues. For one, the ability to upload an image was not available even though I know its supposed to be there. then from the dropdown no matter what option I selected as the description. It would only accept the 'dated student id' option. So I ended up selecting that description and just uploading a picture of my most resent bursar bill that I printed out using my webcam that was so blurry. failed three times. So I ended up auto focusing on the top left corner that showed the date I printed it then it passed even though the rest of the bill was barely visible. Hope that helps. 


So in summary the working solution so far seems to be contacting support. You may get auto closed like I did twice. Just keep doing it. Eventually it wont auto close on you. then after a good 3 week wait someone will contact you. Then you will be able to reapply again for the proper one. But please this time around make sure it says student developer pack somewhere on top of that new application page to make sure you applying to the right on. good luck to anyone in the future that needs this resolution :)
